modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/css/ListVisualizationMode.scss

```diff
@@ -1,13 +1,13 @@
 @import 'cadmin-variables';
 
-html#{$cadmin-selector} .cadmin .table-autofit {
-	.list-visualization-mode-label-cell {
-		min-width: 20ch;
-		width: 30%;
-	}
-
-	.list-visualization-mode-value-cell {
+html#{$cadmin-selector} .cadmin .list-visualization-mode {
+	.field-name {
```

Comment:
`list-visualization-mode` is relevant for tests. `field-name` and `list-section-label` are renames I missed in https://github.com/liferay-frontend/liferay-portal/pull/3990